packages/list_bloc/example/list_bloc_example/README.md

```diff
@@ -0,0 +1,4 @@
+# list_bloc_example
```

Comment:
Let's not create a new folder named `list_bloc_example` instead the project example project be directly inside `example`.